.circleci/config.yml.j2

```diff
@@ -616,6 +644,26 @@ commands:
             .circleci/bin/test-airflow 'ap-airflow' '<< parameters.tag >>'
       - store_test_results:
           path: /tmp/test-reports
+  scan-snyk-docker-image:
+    description: "Snyk: Dependency scanner for a Docker image"
+    parameters:
+      image_name:
+        type: string
+        default: "ap-airflow:2.2.0"
+    steps:
+      - checkout
+      - setup_remote_docker
+      - attach_workspace:
+          at: {{ workspace_prefix }}
+      - run:
+          name: Load archived Docker image
+          command: docker load -i {{ workspace_prefix }}/saved-images/<< parameters.image_name >>-onbuild.tar
+      - snyk/scan:
+          docker-image-name: << parameters.image_name >>-onbuild
```

Comment:
Adding `onbuild` here makes it work but without it, it doesn't. I guess it's because of how we load the archived image. 
Is this enough?